m4_sessions/1710-inning/group_six.md

```diff
@@ -1,18 +1,26 @@
 Group Six
 
-Session Title: "Learning another language"
+Session Title: "Learning a new programming language"
 
-Session Title:Description: Why should students come to this session?
+Session Title:Description: Here at Turing, we all learn how to be awesome developers in Javascript and/or Ruby on Rails. Once you ascend from the basement and enter the real world, your company will probably work in a language/framework you have never used before. That can be pretty intimidating. Come learn about new languages we are excited about, and see how jumping into a new one isn't as scary as it looks.
 
-Students will need to be comfortable learning another language by the time they leave Turing if they want to market themselves as more than just Rails- or JavaScript- developers.
-Session Outcomes: What will students leave with or gain from this session?
+### Session layout:
+* 10 min: "Would you rather?" Icebreaker/Introduction (Everyone)
+* 5 min: Intro with StackOverflow survey statistics (most common languages, average number of languages devs know, charts) (JF)
```

Comment:
In your overview, also provide students clear understanding of why you are going to go over mythical creatures, Elm and Go.